src/main/java/frc/trigon/robot/subsystems/turret/kablamaturret/KablamaTurretConstants.java

```diff
@@ -0,0 +1,59 @@
+package frc.trigon.robot.subsystems.turret.kablamaturret;
+
+import com.ctre.phoenix6.StatusSignal;
+import com.ctre.phoenix6.configs.CANcoderConfiguration;
+import com.ctre.phoenix6.configs.TalonFXConfiguration;
+import com.ctre.phoenix6.hardware.CANcoder;
+import com.ctre.phoenix6.hardware.TalonFX;
+import com.ctre.phoenix6.signals.AbsoluteSensorRangeValue;
+import com.ctre.phoenix6.signals.InvertedValue;
+import com.ctre.phoenix6.signals.NeutralModeValue;
+import com.ctre.phoenix6.signals.SensorDirectionValue;
+
+public class KablamaTurretConstants {
+    private static final int
+            MOTOR_ID = 1,
+            ENCODER_ID = 2;
+    private static final InvertedValue INVERTED_VALUE = InvertedValue.CounterClockwise_Positive;
+    private static final NeutralModeValue NEUTRAL_MODE_VALUE = NeutralModeValue.Brake;
+    private static final AbsoluteSensorRangeValue ENCODER_RANGE = AbsoluteSensorRangeValue.Signed_PlusMinusHalf;
+    private static final SensorDirectionValue ENCODER_DIRECTION = SensorDirectionValue.CounterClockwise_Positive;
+    private static final double ENCODER_OFFSET = 0;
+    private static final CANcoder ENCODER = new CANcoder(ENCODER_ID);
+    static final TalonFX MOTOR = new TalonFX(MOTOR_ID);
+    static final boolean FOC_ENABLED = true;
```

Comment:
this is kinda like a motor setting, so I would advise putting it at the top of the class.